src/components/Column/Column.scss

```diff
@@ -64,6 +64,29 @@
   position: relative;
 }
 
+.column__header-emoji-suggestions {
+  position: absolute;
+  top: calc(100% + $spacing--xs);
+  left: 0;
```

Comment:
After looking into this, it's a bit more tricky than I thought.
- the bug exists with the current note text input as well.
- caused by `scrollIntoView`
https://github.com/inovex/scrumlr.io/blob/6aba47e21147d4ee08c252f7af1713b3b543e8bb/src/components/EmojiSuggestions/EmojiSuggestions.tsx#L22
- usually this shouldn't scroll horizontally when it's not necesarry, but is still does (in firefox at least, chrome isn't affected)

I think only scrolling vertically using `scroll` on the container instead of `scrollIntoView` will fix this, but it'll take a moment to implement.

Reply:
is the scroll necessary in the first place? I mean it always fits on the screen, but if the user keeps it out of range it doesn't have to be the component's responsibility to be actually visible imo

Reply:
the scrolling allows keyboard navigation when the suggestions overflow. Idealy it should only scroll on the parent container and not the columns

Reply:
ahh ok I understand now. It's weird how this problem only affects Firefox. Would be nice if you found a solution though as it's kinda annoying 😅

Reply:
should be fixed with https://github.com/inovex/scrumlr.io/pull/4274/commits/0fc65dd1af33345ea28e089c810cca7fecc534f4